newIDE/app/src/stories/everything-else.stories.js

```diff
@@ -2217,39 +2216,6 @@ storiesOf('SearchPanel', module)
     />
   ));
 
-storiesOf('InstructionSelector', module)
```

Comment:
do we have a story for the InstructionSelector elsewhere? so we can check the ReadOnlyTreeView

Reply:
I removed the InstructionSelector component because it was not used anywhere. I can try to add a story for the InstructionOrObjectSelector